content/collections/account-management/en/account-settings.md

```diff
@@ -44,13 +44,17 @@ Click *View Usage Over Time* in the session capture card to review usage specifi
 From the project list, you can:
 
 - View a list of projects that are capturing replays, along with their sample rate and current instrumentation status.
-- Toggle Session Capture on or off for a specific project. Turning off session capture for a specific project only applies to future sessions.
-Click a project name to modify the sample rate, or percentage of user sessions captured. Modifying the sample rate in settings overrides the existing SDK implementation's sample rate, and may take several minutes to take effect.
+- Toggle *Session Capture* on or off for a specific project. Turning off session capture for a specific project only applies to future sessions.
 
-    From the sampling module, click View User Sessions Chart to view an estimate of the total sessions captured on a monthly basis. 
+    Click a project name to modify the sample rate, or percentage of user sessions captured. Modifying the sample rate in settings overrides the existing SDK implementation's sample rate, and may take several minutes to take effect.
+    
+    ![session_replay_settings.png](/docs/output/img/account-management/session_replay_settings.png)
+
+    From the sampling module, click *View User Sessions Chart* to view an estimate of the total sessions captured on a monthly basis. 
 
 - Click *+ Select Project* from the bottom of the list to add a project. Projects can't be removed once added.
-By default, the project's session capture are set to `on`, and its sample rate matches the rate established in your SDK implementation. Adding a project via the SDK doesn't automatically add it to the list. However, an admin adding a new organization via the organizational settings adds a project to the list with session capture set to on and a sample rate of 100%.
+
+     By default, the project's session capture are set to `on`, and its sample rate matches the rate established in your SDK implementation. Adding a project via the SDK doesn't automatically add it to the list. However, an admin adding a new organization via the organizational settings adds a project to the list with session capture set to on and a sample rate of 100%.
```

Comment:
```suggestion
     By default, the project's session capture is set to `on`, and its sample rate matches the rate set in your SDK implementation. Adding a project with the SDK doesn't automatically add it to the list. However, an admin adding a new organization through the organizational settings adds a project to the list with session capture set to on and a sample rate of 100%.
```

This whole paragraph reads kind of weird to me. Can you take a stab at cleaning it up?